### PR TITLE
Add footer

### PR DESCRIPTION
**Pull request by kc-si**
Create _footer.html.erb layout, add footer styleshet, update application.html.erb and configure precompilation.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,12 +8,26 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+
+    <%= stylesheet_link_tag "sticky-footer-navbar" %>
```
Comment (marekciupak) [inline app/views/layouts/application.html.erb:12]: Here you are importing other CSS files:

https://github.com/kc-si/JSCLook/blob/d5f85db828a57858bd02f5e91129af38a48d2c68/app/assets/stylesheets/application.bootstrap.scss#L1-L2

Maybe it is possible to import `sticky-footer-navbar.css` the same way?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -8,12 +8,26 @@
...
     <%= yield %>
+
+    <footer class="footer container-fluid">
+      <%= render 'layouts/footer' %>
+    </footer>
+
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" crossorigin="anonymous">
+    </script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.14.7/dist/umd/popper.min.js" crossorigin="anonymous">
+    </script>
```
Comment (marekciupak) [inline app/views/layouts/application.html.erb:29]: I think you may be able to install them using NPM:

```shell
npm install --save jquery popper.js
```

and maybe you won't need to add them manually here :)

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -8,12 +8,26 @@
...
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/js/bootstrap.min.js" crossorigin="anonymous">
+    </script>
```
[the remark this answers is not in the document]
Comment (kc-si) [reply app/views/layouts/application.html.erb:40]: Yes I thought the same, but it is needed. I think, maybe installation of bootstrap give us only css styles?

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (kc-si) [reply app/views/layouts/application.html.erb:38]: I install them and that doesn't work :(

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (kc-si) [reply app/views/layouts/application.html.erb:12]: You mean to use importmap and import from internet? or from my localization but just this method?

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (kc-si) [reply app/views/layouts/application.html.erb:38]: I try use importmap as well, not work, but in the importmap I'm not sure, maybe needs some more to set like javascript controllers?

**Pull request merged**